src/components/Pagination.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Pagination as AntPagination } from "antd";
+import { useHistory, useLocation } from "react-router-dom";
+import queryString from "query-string";
+
+interface Props {
+  totalResults: number;
+  value: number;
+  pageSize: number;
+  dataTestId?: string;
+}
+
+export const Pagination: React.FC<Props> = ({
+  totalResults,
+  value,
+  pageSize,
+  dataTestId,
+}) => {
+  const { replace } = useHistory();
+  const { search, pathname } = useLocation();
+  const setPage = (page: number) =>
+    replace(
+      `${pathname}?${queryString.stringify(
+        {
+          ...queryString.parse(search, { arrayFormat }),
+          page,
+        },
+        { arrayFormat }
+      )}`
```

Comment:
im wondering if this is something we might want to pull out and turn into more of a global utility function. I've seen similar logic used in other parts of the app.

Reply:
sure, this could be a hook